packages/host/src/routers/CatalogueRouter.tsx

```diff
@@ -13,10 +12,12 @@ const fullItemPath = RouteBuilder.create(AppRoute.Catalogue)
   .build();
 
 export const CatalogueRouter: FC = () => {
-  if (useMatch(fullItemPath)) {
+  const gotoItems = useMatch(fullItemPath);
```

Comment:
I changed all the routers which were using multiple `useMatch` - I started to get a react error that the number of hooks being called were changing during renders, which makes sense, but it wasn't showing up before - now it was, so just made sure the component called all the hooks every render